src/pages/api/organize/[orgId]/journeys/[journeyId]/getTagMetadata.tsx

```diff
@@ -0,0 +1,30 @@
+import { uniqBy } from 'lodash';
+
+import { ZetkinJourneyInstance, ZetkinTag } from 'types/zetkin';
+
+export interface TagMetadata {
+  groups: ZetkinTag['group'][];
+  valueTags: ZetkinJourneyInstance['tags'];
+}
+
+export const getTagMetadata = (
```

Comment:
I like how simple this function is. I think it's a great candidate for a couple of unit tests.

Reply:
done